### PR TITLE
add additinal rules for haproxy

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

File: Containers/docker-socket-proxy/haproxy.cfg [hunk 1]
```diff
@@ -11,6 +11,8 @@ frontend http
     http-request deny unless { src 127.0.0.1 } || { src ::1 } || { src NC_IPV4_PLACEHOLDER } || { src NC_IPV6_PLACEHOLDER }
     http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/containers/nc_app_[a-zA-Z0-9_.-]+/((json)|(start)|(stop)) } METH_GET
     http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/containers/nc_app_[a-zA-Z0-9_.-]+ } METH_DELETE
+    http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/containers/nc_app_[a-zA-Z0-9_.-]+/((json)) } METH_GET
```
Comment (szaimen) [inline Containers/docker-socket-proxy/haproxy.cfg:14]: isnt this covered by the rule two lines above? or should it actually be meth_post? then we should probably also change the rule below and just include the json there?

**Comment by bigcat88** [comment]
List of endpoints currently using AppEcosystem:
```
post containers/create?name=%s

post containers/%s/start

post containers/%s/stop

delete containers/%s

post images/create?fromImage=%s

get containers/%s/json

post volumes/create

delete volumes/%s
```

Yep, rules needs adjustments.

**Comment by bigcat88** [comment]
@andrey18106 Is there anything else you'd like to include in the pull request from your end?

**Review by bigcat88 — APPROVED** [review]

**Comment by andrey18106** [comment]
> @andrey18106 Is there anything else you'd like to include in the pull request from your end?

Seems like no. Additional restriction of volume creation DriverOpts.device option usage looks good to me.

**Review by andrey18106 — APPROVED** [review]

**Pull request merged**